src/Tasks/Microsoft.NET.Build.Tasks.NETFramework/Microsoft.NET.Build.Tasks.NETFramework.csproj

```diff
@@ -0,0 +1,111 @@
+  </PropertyGroup>
+
+  <ItemDefinitionGroup>
+    <PackageReference>
+      <PrivateAssets>All</PrivateAssets>
+      <Publish>true</Publish>
+    </PackageReference>
+  </ItemDefinitionGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Build.Framework" Version="$(MsBuildPackagesVersion)" ExcludeAssets="Runtime" />
+    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="$(MsBuildPackagesVersion)" ExcludeAssets="Runtime" />
+    <PackageReference Include="System.Reflection.Metadata" Version="$(SystemReflectionMetadataVersion)" Condition=" '$(TargetFramework)' == 'netcoreapp1.0' " />
+    <PackageReference Include="System.Diagnostics.FileVersionInfo" Version="4.0.0" Condition=" '$(TargetFramework)' == 'netcoreapp1.0' " />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Include="..\Microsoft.NET.Build.Tasks\BuildErrorException.cs">
+      <Link>BuildErrorException.cs</Link>
+    </Compile>
+    <Compile Include="..\Microsoft.NET.Build.Tasks\ConflictResolution\*.cs">
+      <Link>ConflictResolution\%(FileName)%(Extension)</Link>
+    </Compile>
+    <Compile Include="..\Microsoft.NET.Build.Tasks\DiagnosticsHelper.cs">
+      <Link>DiagnosticsHelper.cs</Link>
+    </Compile>
+    <Compile Include="..\Microsoft.NET.Build.Tasks\DiagnosticMessageSeverity.cs">
+      <Link>DiagnosticMessageSeverity.cs</Link>
+    </Compile>
+    <Compile Include="..\Microsoft.NET.Build.Tasks\ItemUtilities.cs">
+      <Link>ItemUtilities.cs</Link>
+    </Compile>
+    <Compile Include="..\Microsoft.NET.Build.Tasks\GetDependsOnNETStandard*.cs">
+      <Link>%(FileName)%(Exetnsion)</Link>
+    </Compile>
+    <Compile Include="..\Microsoft.NET.Build.Tasks\MetadataKeys.cs">
+      <Link>MetadataKeys.cs</Link>
+    </Compile>
+    <Compile Include="..\Microsoft.NET.Build.Tasks\NuGetUtils.cs">
+      <Link>NuGetUtils.cs</Link>
+    </Compile>
+    <Compile Include="..\Microsoft.NET.Build.Tasks\TaskBase.cs">
+      <Link>TaskBase.cs</Link>
+    </Compile>
+    <EmbeddedResource Include="..\Microsoft.NET.Build.Tasks\Resources\Strings.resx">
+      <Link>Resources\%(FileName)%(Extension)</Link>
+      <Generator>ResXFileCodeGenerator</Generator>
+      <LastGenOutput>Strings.Designer.cs</LastGenOutput>
+      <CustomToolNamespace>Microsoft.NET.Build.Tasks</CustomToolNamespace>
+    </EmbeddedResource>
+    <EmbeddedResource Include="..\Microsoft.NET.Build.Tasks\Resources\Strings.*.resx">
+      <Link>Resources\%(FileName)%(Extension)</Link>
+      <DependentUpon>Strings.resx</DependentUpon>
+    </EmbeddedResource>
+    <Compile Include="..\Microsoft.NET.Build.Tasks\Resources\Strings.Designer.cs">
+      <Link>Resources\%(FileName)%(Extension)</Link>
+      <DesignTime>true</DesignTime>
+      <AutoGen>true</AutoGen>
+      <DependentUpon>Strings.resx</DependentUpon>
+    </Compile>
+
+  </ItemGroup>
+
+  <Target Name="PrepareAdditionalFilesToLayout" BeforeTargets="AssignTargetPaths">
```

Comment:
Instead of having any of the package layout stuff in this project, we probably want building the other project to include the layout from this one.